Assets/Scripts/Model/Content/SecondEdition/Upgrades/Talent/Outmaneuver.cs

```diff
@@ -25,7 +25,7 @@ public class OutmaneuverAbilitySE : Abilities.FirstEdition.OutmaneuverAbility
     {
         protected override bool AbilityCanBeUsed()
         {
-            if (!Combat.ShotInfo.InArcByType(Arcs.ArcType.Front)) return false;
+            if (!Combat.ChosenWeapon.WeaponInfo.ArcRestrictions.Contains(Arcs.ArcType.Front)) return false;
```

Comment:
I haven't looked at these Objects, but my gut says you'll need two checks.

1. Is the defender in front arc of the attacker?
2. Does the chosen weapon have the ability to attack through that arc?

I'm gonna go look at that now, and edit this comment if it looks like you've already covered those cases.

Reply:
@nickell-andrew agreed; I originally kept the original line to ensure the first condition (defender is in front arc of attacker).  then it occurred to me that if the chosen weapon is a front arc weapon, the defender must be already be in the front arc of the attacker (because if it isn't, the weapon wouldn't have a shot and we wouldn't reach this code).  so it didn't seem necessary to me to check yet again that the defender is in the front arc of the attacker.

anyway, that was my thought process.  does that sound right to you?

Reply:
As I remember code, when attack is started, separate decisions for each weapon + arc is shown (For example, in case of Shadow Caster, Primary + Front and Primary + left-sided Mobile Arc). So, I think that we need only to get access to this stored decision.

Reply:
Can a weapon contain more than just a front arc, though? This may not be relevant now as all front arc weapons that I know of don't also contain rear arcs etc, but that may not be the case forever.

My brain was a bit jellied, so I'll update my cases to:

1. Is the attacking weapon capable of performing (V) attacks?
2. Is the defender in the attacker's front arc?
3. Does the defender have no weapons whose firing arcs include the attacker?

Reply:
@nickell-andrew I believe the third case is already checked by the existing code, in the lines just below my changes.

I'll re-add the original check so all 3 cases are explicitly checked, and see what you guys think.

Reply:
If ship has, for example, front and rear arcs for primary weapon, then, in code of Fly Casual, that ship has two separate primary weapons (like forward with firepower 3, and rear with firepower 2)

Reply:
This makes sense to me. The first check is excluding turret attacks and bullseye attacks, the second ensures that the defender is indeed in front arc (probably redundant with the first check per @Sandrem) and line 32-33 checks that the attacker isn't in arc of the defender. Seems good to me.